ui/app/components/send_/send-footer/send-footer.utils.js

```diff
@@ -0,0 +1,88 @@
+const ethAbi = require('ethereumjs-abi')
+const ethUtil = require('ethereumjs-util')
+const { TOKEN_TRANSFER_FUNCTION_SIGNATURE } = require('../send.constants')
+
+function formShouldBeDisabled ({ inError, selectedToken, tokenBalance, gasTotal }) {
```

Comment:
I kinda see utils as being very generic functions that have no knowledge of the view, so I'm not too sure about having this function here. I'd rather this be in the component.